launcher/ui/pages/instance/ManagedPackPage.cpp

```diff
@@ -471,8 +489,7 @@ void FlameManagedPackPage::parseManagedPack()
 
 QString FlameManagedPackPage::url() const
 {
-    // FIXME: We should display the websiteUrl field, but this requires doing the API request first :(
-    return {};
+    return "https://www.curseforge.com/projects/" + m_inst->getManagedPackID();
```

Comment:
Perhaps leave the FIXME here, as CurseForge could change their URL structure in the future.

Reply:
unlikely to happen, but just to be sure I added it back